docs/source/getting_started/benchmarks.rst

```diff
@@ -54,8 +60,8 @@ We make the following observations running the benchmark:
 
 - SimCLR and SimSiam benefit from larger batch sizes. MoCo (which uses a
   memorybank) performs very well also for smaller batch sizes.
-- All models need roughly 3h to complete the 200 epoch benchmark and use roughly
-  the same amount of GPU memory.
+- All models need roughly 3h (on a V100 GPU) to complete the 200 epoch benchmark 
+  and use roughly the same amount of GPU memory.
```

Comment:
2x roughly, replace the second one by approximately

Reply:
and also say how much memory they need

Reply:
The memory is reported in the table